druid-shell/src/mac/mod.rs

```diff
@@ -145,13 +163,47 @@ impl WindowBuilder {
     }
 }
 
+struct WindowClass(*const Class);
+unsafe impl Sync for WindowClass {}
+
+lazy_static! {
+    static ref WINDOW_CLASS: WindowClass = unsafe {
+        let mut decl =
+            ClassDecl::new("DruidWindow", class!(NSWindow))
+            .expect("Window class defined");
+
+        decl.add_ivar::<BOOL>("acceptsMouseMove");
```

Comment:
I'm not sure I fully understand the motivation for this variable. I've been studying other code bases and it seems like the best practice for enabling mouse movement is to set up an NSTrackingRegion. Examples in [minifb](https://github.com/emoon/rust_minifb/blob/master/src/native/macosx/OSXWindowFrameView.m#L88) and [rutabaga](https://github.com/wrl/rutabaga/blob/master/src/platform/cocoa/window.m#L108) linked. I haven't studied it hard enough to know the true best way.

Reply:
I have no idea what the "best" way is, this is more the only way I found to do when I needed to last time. I'll take a look at those examples if you think that way is better.

The major dislike I have of the way I've done it here is the need for the extra subclass of NSWindow